numpy/polynomial/polyutils.py

```diff
@@ -735,7 +734,7 @@ def format_float(x, parens=False):
 
     exp_format = False
     if x != 0:
-        a = absolute(x)
+        a = np.abs(x)
```

Comment:
`np.abs` is `np.absolute` which is imported from `np._core -> np._core.numeric -> np._core.umath`.